auction9-frontend/src/app/add-update-auction-dialog/add-update-auction-dialog.component.html

```diff
@@ -110,11 +114,14 @@ <h1 mat-dialog-title>
     </mat-icon>
     <ngx-material-timepicker #endtimepicker>
     </ngx-material-timepicker>
-    <mat-error *ngIf="addAuctionForm.controls.endTime.hasError('required')">
+    <mat-error *ngIf="addOrUpdateAuctionForm.controls.endTime.hasError('required')">
       You have to pick end time
     </mat-error>
   </mat-form-field>
-  <mat-error *ngIf="addAuctionForm.hasError('startTimeLessThanEndTime')">
+  <mat-error *ngIf="addOrUpdateAuctionForm.hasError('startTimeLessThanEndTime')">
+    End time must be greater than start time
+  </mat-error>
+  <mat-error *ngIf="addOrUpdateAuctionForm.hasError('minTimeGreaterThanNow')">
     End time must be greater than start time
```

Comment:
It looks like wrong message info?

Reply:
I forgot to remove that line, it was part of custom validator that I'll create later